personio-personnel-data-api-oa3.yaml

```diff
@@ -43,6 +43,23 @@ paths:
         - BearerAuth: []
       tags:
         - Employees
+      parameters:
+        - name: limit
+          in: query
+          required: false
+          description: Pagination attribute to limit the number of employees returned per page.
+          schema:
+            type: integer
+            minimum: 1
+            default: 200
+        - name: offset
```

Comment:
@trung-personio after looking into the implementation, this parameter should be be named `page`. Because naming it as an offset means we have to calculate it for every next request, like offset = max(0, (page - 1) * limit). And that is why offset zero and offset 1 have the same result. So either we change how it is implemented or we change the parameter names so it will be more clear that it is a page number. What do you think?

Reply:
@hsalem7, 

> Because naming it as an offset means we have to calculate it for every next request
I'm not really sure what you meant here, `offset` behaves similar as `page` for the time being. The parameter name `offset` brings the confusion, it is actually a page instead of an offset. To be honest I don't really like that name but as it was implemented in other APIs, then I did not want to bring the exception here.